package.json

```diff
@@ -12,11 +12,12 @@
   },
   "private": true,
   "scripts": {
-    "init-local-environment": "node scripts/init-local-environment",
-    "postinstall": "postinstall && npm-run-all --silent build:tslint-rules build:schematics synchronize-lazy-components init-local-environment",
+    "init-development-environment": "node scripts/init-development-environment",
```

Comment:
I honestly think that we should keep the `init-local-environment` as script entry here. Helps the out-of-a-habit working developer. What do you guys think?

Reply:
I would not keep the old script to make developers with a habit aware of this breaking change. `npm` even helps you with that.

![image](https://user-images.githubusercontent.com/50741106/127671702-cb86a683-b141-40f9-b64e-6919cbba2931.png)